### PR TITLE
Lists the Slack relevant channels and apps

### PR DESCRIPTION
**Pull request by daniloaliberti**
New iOS joiners can easily find a list of Slack relevant channels and apps.

Conflicting with [other PR.](https://github.com/Babylonpartners/ios-playbook/pull/47)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,35 @@ Prior to starting, make sure you have a Babylon GitHub account and that you have
 - [ios-build-distribution](https://github.com/Babylonpartners/ios-build-distribution)
 - [ios-fastlane-match](https://github.com/Babylonpartners/ios-fastlane-match)
 
+On Babylon's Slack, check the channels iOS Team related channels:
```
Comment (RuiAAPeres) [inline README.md]: I would re-phrase this to `As an iOS Engineer, you should be in the following Slack channels`.

File: README.md [hunk 2]
```diff
@@ -82,6 +82,35 @@ Prior to starting, make sure you have a Babylon GitHub account and that you have
...
+- ios-questions
+- ios-sdk
+- ios-oss
+- ios-recruiting
+
+Others
+- demo_frontend
+- developers
+- general
+- announcements
```
Comment (RuiAAPeres) [inline README.md]: Mentioning `general` and `announcements ` is redundant because by default you are there.

File: README.md [hunk 3]
```diff
@@ -82,6 +82,35 @@ Prior to starting, make sure you have a Babylon GitHub account and that you have
...
+
+Apps
+- pull-reminders
+- peakon
+
+And also your squad will have
```
Comment (RuiAAPeres) [inline README.md]: I would re-phrase this: `Make sure you join your Tribe/Squad's Slack channels.`

Comment (daniloaliberti) [reply README.md]: https://github.com/Babylonpartners/ios-playbook/pull/49/commits/2afbc132f6db2fd0b9ef2818b29c8d99c46949c4

File: README.md (hunk 2 above)
Comment (daniloaliberti) [reply README.md]: https://github.com/Babylonpartners/ios-playbook/pull/49/commits/2afbc132f6db2fd0b9ef2818b29c8d99c46949c4

File: README.md (hunk 1 above)
Comment (daniloaliberti) [reply README.md]: https://github.com/Babylonpartners/ios-playbook/pull/49/commits/2afbc132f6db2fd0b9ef2818b29c8d99c46949c4

**Pull request merged**